#! /bin/sh /usr/share/dpatch/dpatch-run
## 16-rename-tao-programs.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## DP: make tao program names quite similar.

@DPATCH@
--- ACE_wrappers/TAO/orbsvcs/Event_Service/Event_Service.mpc-	2009-10-20 14:10:53.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/Event_Service/Event_Service.mpc	2010-05-12 23:38:30.548383762 +0200
@@ -2,5 +2,5 @@
 // $Id: Event_Service.mpc 87176 2009-10-20 12:10:53Z johnnyw $
 
 project : orbsvcsexe, install, rtevent_serv, rtschedevent, bidir_giop, svc_utils {
-  exename  = Event_Service
+  exename  = tao_rtevent
 }
--- ACE_wrappers/TAO/orbsvcs/Time_Service/Time_Service.mpc-	2008-09-23 23:50:40.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/Time_Service/Time_Service.mpc	2010-05-12 23:35:55.165781003 +0200
@@ -2,7 +2,7 @@
 // $Id: Time_Service.mpc 82815 2008-09-23 21:50:40Z jtc $
 
 project(Time_Service_Server) : orbsvcsexe, install, time_serv, naming, utils, avoids_corba_e_micro {
-  exename  = Time_Service_Server
+  exename  = tao_costime_server
   Source_Files {
     Server_i.cpp
     Time_Service_Server.cpp
@@ -10,7 +10,7 @@
 }
 
 project(Time_Service_Clerk) : orbsvcsexe, install, time_serv, naming, utils, avoids_corba_e_micro {
-  exename  = Time_Service_Clerk
+  exename  = tao_costime_clerk
   Source_Files {
     Clerk_i.cpp
     Time_Service_Clerk.cpp
--- ACE_wrappers/TAO/orbsvcs/Logging_Service/Event_Logging_Service/Event_Logging_Service.mpc-	2007-01-03 17:08:20.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/Logging_Service/Event_Logging_Service/Event_Logging_Service.mpc	2010-05-12 23:41:39.960399745 +0200
@@ -1,5 +1,5 @@
 // $Id: Event_Logging_Service.mpc 76300 2007-01-03 16:08:20Z elliott_c $
 
 project: namingexe, iortable, install, dseventlogadmin_serv, valuetype {
-  exename   = Event_Logging_Service
+  exename   = tao_tls_event
 }
--- ACE_wrappers/TAO/orbsvcs/Logging_Service/RTEvent_Logging_Service/RTEvent_Logging_Service.mpc-	2007-01-03 17:08:20.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/Logging_Service/RTEvent_Logging_Service/RTEvent_Logging_Service.mpc	2010-05-12 23:42:38.236383274 +0200
@@ -1,5 +1,5 @@
 // $Id: RTEvent_Logging_Service.mpc 76300 2007-01-03 16:08:20Z elliott_c $
 
 project: namingexe, iortable, install, rteventlogadmin_serv, valuetype {
-  exename   = RTEvent_Logging_Service
+  exename   = tao_tls_rtevent
 }
--- ACE_wrappers/TAO/orbsvcs/Logging_Service/Basic_Logging_Service/Basic_Logging_Service.mpc-	2007-01-03 17:08:20.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/Logging_Service/Basic_Logging_Service/Basic_Logging_Service.mpc	2010-05-12 23:40:57.948383480 +0200
@@ -1,5 +1,5 @@
 // $Id: Basic_Logging_Service.mpc 76300 2007-01-03 16:08:20Z elliott_c $
 
 project: namingexe, iortable, install, dslogadmin_serv, valuetype {
-  exename   = Basic_Logging_Service
+  exename   = tao_tls_basic
 }
--- ACE_wrappers/TAO/orbsvcs/Logging_Service/Notify_Logging_Service/Notify_Logging_Service.mpc-	2007-01-03 17:08:20.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/Logging_Service/Notify_Logging_Service/Notify_Logging_Service.mpc	2010-05-12 23:42:10.168382105 +0200
@@ -1,5 +1,5 @@
 // $Id: Notify_Logging_Service.mpc 76300 2007-01-03 16:08:20Z elliott_c $
 
 project: namingexe, iortable, install, dsnotifylogadmin_serv, valuetype {
-  exename   = Notify_Logging_Service
+  exename   = tao_tls_notify
 }
--- ACE_wrappers/TAO/orbsvcs/Naming_Service/Naming_Service.mpc-	2009-10-19 14:18:00.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/Naming_Service/Naming_Service.mpc	2010-05-12 23:37:10.200383656 +0200
@@ -2,7 +2,7 @@
 // $Id: Naming_Service.mpc 87156 2009-10-19 12:18:00Z johnnyw $
 
 project : orbsvcsexe, install, naming_serv, imr_client, svc_utils {
-  exename  = Naming_Service
+  exename  = tao_cosnaming
   Source_Files {
     Naming_Server.cpp
     Naming_Service.cpp
--- ACE_wrappers/TAO/orbsvcs/Trading_Service/Trading_Service.mpc-	2007-01-03 17:08:20.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/Trading_Service/Trading_Service.mpc	2010-05-12 23:36:36.917383940 +0200
@@ -2,6 +2,6 @@
 // $Id: Trading_Service.mpc 76300 2007-01-03 16:08:20Z elliott_c $
 
 project : orbsvcsexe, utils, install, naming, trading_serv {
-  exename  = Trading_Service
+  exename  = tao_costrading
 }
 
--- ACE_wrappers/TAO/orbsvcs/Notify_Service/Notify_Service.mpc-	2009-10-29 00:29:32.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/Notify_Service/Notify_Service.mpc	2010-05-12 23:39:53.260409652 +0200
@@ -14,7 +14,7 @@
 }
 
 project(Notify_Service) : orbsvcsexe, install, tao_notify_service, notify_monitor_control {
-  exename  = Notify_Service
+  exename  = tao_cosnotification
   Source_Files {
     Notify_Server.cpp
   }
--- ACE_wrappers/TAO/orbsvcs/Concurrency_Service/Concurrency_Service.mpc-	2009-10-20 14:10:53.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/Concurrency_Service/Concurrency_Service.mpc	2010-05-12 23:13:25.228530177 +0200
@@ -2,5 +2,5 @@
 // $Id: Concurrency_Service.mpc 87176 2009-10-20 12:10:53Z johnnyw $
 
 project : orbsvcsexe, install, concurrency_serv, naming, utils, svc_utils {
-  exename  = Concurrency_Service
+  exename  = tao_cosconcurrency
 }
--- ACE_wrappers/TAO/orbsvcs/CosEvent_Service/CosEvent_Service.mpc-	2007-01-03 17:08:20.000000000 +0100
+++ ACE_wrappers/TAO/orbsvcs/CosEvent_Service/CosEvent_Service.mpc	2010-05-12 23:12:44.184952436 +0200
@@ -2,5 +2,5 @@
 // $Id: CosEvent_Service.mpc 76300 2007-01-03 16:08:20Z elliott_c $
 
 project : orbsvcsexe, install, naming, event_serv, svc_utils, iortable, messaging_optional, portableserver {
-  exename  = CosEvent_Service
+  exename  = tao_cosevent
 }
--- ACE_wrappers/TAO/orbsvcs/LifeCycle_Service/LifeCycle_Service.mpc-	2008-09-23 23:50:40.000000000 +0200
+++ ACE_wrappers/TAO/orbsvcs/LifeCycle_Service/LifeCycle_Service.mpc	2010-05-13 11:32:27.776383749 +0200
@@ -2,5 +2,5 @@
 // $Id: LifeCycle_Service.mpc 82815 2008-09-23 21:50:40Z jtc $
 
 project : orbsvcsexe, lifecycle_skel, naming, trading_serv, avoids_minimum_corba, utils, install {
-  exename  = LifeCycle_Service
+  exename  = tao_coslifecycle
 }
